code/modules/virus2/centrifuge.dm

```diff
@@ -457,6 +457,9 @@
 
 			var/antibody = antibody_choices[choice]
 
+			if (antibodies[antibody] < 49)
```

Comment:
<=, or just <50

Reply:
I put 49 since I considered a 2 minutes wait to still be acceptable.

Reply:
But your feedback text recommends 50

Reply:
pulling the vial out to get another sample at concentration 50 may as well take an extra minute anyway. bother aren't exclusive.